core/consensus/grandpa/structs.hpp

```diff
@@ -129,7 +128,7 @@ namespace kagome::consensus::grandpa {
   struct GrandpaJustification {
     RoundNumber round_number;
     BlockInfo block_info;
-    std::vector<SignedPrecommit> items;
+    std::vector<SignedMessage> items;
```

Comment:
I loved previous name, as it reflects the fact that Justification is vector of Signed precommits. Can we keep it somehow?

Reply:
Unfortunately not. CatchUpResponse contains two array: prevotes and precommits. This is reason to make it universal.